test_wrapper.py

```diff
@@ -256,3 +256,8 @@ def test_load_lazy_nested(self):
         self.assertIsNone(res['a'])
         self.assertIsNone(res['test1']['b'])
         self.assertIsNone(res['test2']['test3']['c'])
+
+    def test_write_unicode(self):
+        l = ['a', 'b']
```

Comment:
This can be written in one line with
```python
l = [u'a', u'b']
```
or is there a reason why you do it in two steps?